extensions/binding/org.eclipse.smarthome.binding.ble.bluegiga/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: BlueGiga BLE Bridge Binding Fragment
+Bundle-SymbolicName: org.eclipse.smarthome.binding.ble.bluegiga;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/com.zsmartsystems.bluetooth.bluegiga-1.0.0-SNAPSHOT.jar
+Fragment-Host: org.eclipse.smarthome.binding.ble
+Import-Package: 
+ gnu.io;version="3.12.0.OH",
```

Comment:
That's a good point. We indeed do not have gnu.io available in ESH for license (GPL) reasons. We therefore cannot add it to our 3rd party p2 site for the target platform to pick it up.
We can check if we can create a works-with CQ that would at least allow us to compile against it (while not distributing it). The other option would be to add the bluegiga fragment simply to openhab2-addons and stick with the DBUS solution in ESH.